### DISCUSSION TITLE
AUTOCAD MEP

### DISCUSSION THREAD

**Discussion by PASQUETO-CODE** [globallightingdata/gldf#16] [General]
Bonjour à vous, 
Je voulais savoir si le format GLDF pourra remplacer le fichier RFA pour être utiliser dans les maquettes BIM
Merci

**Comment by holg** [comment]
Bonjour à toi,
i do not think Autodesk has the intention to include this file format natively.
There is a third-party (paid) solution, which will support GLDF.

https://relux.com/en/reluxcad-revit.html
